Cargo.toml

```diff
@@ -40,6 +40,7 @@ wasm-bindgen-macro = { path = "crates/macro", version = "=0.2.76" }
 serde = { version = "1.0", optional = true }
 serde_json = { version = "1.0", optional = true }
 cfg-if = "1.0.0"
+num-traits = { version = "0.2.14", optional = true }
```

Comment:
This is a small enough addition could this be a method instead of a trait implementation?